src/components/Practice/PhrasesInAction.jsx

```
@@ -5318,6 +5313,54 @@ const PhrasesInAction = ({
+                              cursor: "pointer",
+                            }}
+                          >
+                            {/* Kannada Letter Box */}
+                            <Box
+                              sx={{
+                                backgroundColor: "#FEBC2F66",
+                                borderRadius: "4px",
+                                //width: "100px",
+                                //height: "100px",
+                                padding: "5px",
+                                display: "flex",
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)